modules/integration/tests-integration/tests-backend/src/test/java/org/wso2/am/integration/tests/throttling/NewThrottleTiersAndLoadOldTiersTestCase.java

```diff
@@ -0,0 +1,610 @@
+
+        //remove created apis
+        clean();
+
+        //add old tiers groups
+        resourceAdminServiceClient.updateTextContent(TIER_XML_REG_CONFIG_LOCATION, newTiersXML);
+        resourceAdminServiceClient.updateTextContent(TIER_XML_REG_CONFIG_APP_LOCATION, newTiersXML);
+        resourceAdminServiceClient.updateTextContent(TIER_XML_REG_CONFIG_RES_LOCATION, newTiersXML);
+
+        APICreationRequestBean apiRequestBean = new APICreationRequestBean(OLD_TIER_TEST_API, OLD_TIER_TEST_API_CONTEXT,
+                API_VERSION_1_0_0, providerName, new URL(apiEndPointUrl));
+        apiRequestBean.setTags(API_TAGS);
+        apiRequestBean.setDescription(API_DESCRIPTION);
+
+        //create and publish a api
+        After_TierTEST_ID = new APIIdentifier(providerName, OLD_TIER_TEST_API, API_VERSION_1_0_0);
+        createAndPublishAPI(After_TierTEST_ID, apiRequestBean, apiPublisherClientUser, false);
+
+        //check the availability of old throttle tiers
+        HttpResponse tierManagePageHttpResponse = apiPublisherClientUser
+                .getAPIManagePage(OLD_TIER_TEST_API, providerName, API_VERSION_1_0_0);
+        assertEquals(tierManagePageHttpResponse.getResponseCode(), HTTP_RESPONSE_CODE_OK,
+                "Response code mismatched when invoke to get Tier Permission Page");
+        assertTrue(tierManagePageHttpResponse.getData().contains(TIER_MANAGE_PAGE_TIER_GOLD),
+                "default tier Gold is not available in Tier Permission page before  add new tear in tiers.xml");
+        assertTrue(tierManagePageHttpResponse.getData().contains(TIER_MANAGE_PAGE_RESOURCE_TIER_SILVER),
+                "default Resource tier Silver is not available in Tier Permission page before  add new tear in "
+                        + "tiers.xml");
+
+        HttpResponse applicationPageHttpResponse = apiStoreClientUser.getApplicationPage();
+        assertEquals(applicationPageHttpResponse.getResponseCode(), HTTP_RESPONSE_CODE_OK,
+                "Response code mismatched when invoke to get Tier Permission Page");
+        assertTrue(applicationPageHttpResponse.getData().contains(TIER_MANAGE_PAGE_APPLICATION_TIER_GOLD),
+                "default tier Gold is not available in Tier Permission page before  add new tear in tiers.xml");
+    }
+
+    private void clean() throws Exception {
+        //remove the Apis
+        apiStoreClientUser.removeApplication(API_TIER_TEST_APP);
+        apiStoreClientUser.removeApplication(APP_TIER_TEST_APP);
+        apiStoreClientUser.removeApplication(RES_TIER_TEST_APP);
+
+        //remove applications
+        deleteAPI(APITierTEST_ID, apiPublisherClientUser);
+        deleteAPI(APPTierTEST_ID, apiPublisherClientUser);
+        deleteAPI(RESTierTEST_ID, apiPublisherClientUser);
+        deleteAPI(Before_TierTEST_ID, apiPublisherClientUser);
+    }
+
+    @AfterClass(alwaysRun = true)
+    public void destroy() throws Exception {
+
+        deleteAPI(After_TierTEST_ID, apiPublisherClientUser);
```

Comment:
No need to delete the api if  super.cleanUp(); is called afterwards . super.cleanUp() will delete all the apis